spec/models/administrative_report_or_publication_spec.rb

```diff
@@ -0,0 +1,9 @@
+# Generated via
```

Comment:
A good start for covering the models would be to check their indexers and human readable type.